src/bleaksport/mux_base.py

```
@@ -167,49 +171,50 @@ async def _run_device(self, addr: str, roles: set[str]) -> None:
             except BleakError as e:
                 msg = str(e)
                 if INPROGRESS_RE.search(msg):
+                    needs_bluez_fallback = True
                     await asyncio.sleep(1.5)
             except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)